src/main/java/org/broadinstitute/hellbender/tools/exome/AllelicCNV.java

```diff
@@ -258,11 +254,99 @@ protected void runPipeline(final JavaSparkContext ctx) {
         final File finalModeledSegmentsFileAsGatkCNV = new File(outputPrefix + "-" + FINAL_SEG_FILE_TAG + "." + GATK_SEG_FILE_TAG + ".seg");
         modeller.writeModeledSegmentFile(finalModeledSegmentsFileAsGatkCNV);
 
-        // Write file for ACS- compatible output to help Broad CGA
+        //write file for ACS-compatible output to help Broad CGA
         final File finalACSModeledSegmentsFile = new File(outputPrefix + "-" + FINAL_SEG_FILE_TAG + "." + CGA_ACS_SEG_FILE_TAG + ".seg");
         modeller.writeAllelicCapSegFile(finalACSModeledSegmentsFile);
 
         ctx.setLogLevel(originalLogLevel);
         logger.info("SUCCESS: Allelic CNV run complete for sample " + sampleName + ".");
     }
-}
+
+    //merge copy-neutral and uncalled segments (unless disabled) and fix up target-segment start breakpoints
+    private List<SimpleInterval> prepareTargetSegments(final Genome genome,
+                                                       final List<ModeledSegment> targetSegmentsWithCalls) {
+        final List<SimpleInterval> targetSegmentsUnfixed;
```

Comment:
If you can do it without too long a variable name, try to capture what needs to be fixed eg `targetSegmentsWithUnfixedBreakpoints`.